.github/pull_request_template.md

```diff
@@ -0,0 +1,59 @@
+## Summary
+
+> 🎯 **Purpose**: Describe the objective of your changes in this Pull-Request.
+> 📜 **Example Usage**:  If adding new functionality, include a small code snippet that demonstrates the the end-to-end usage of your new methods.
+> 📄 **Output**: If possible, share the exact output from this ccode (print out any returned variables).
```

Comment:
I feel this (`output`) should go to another section below, such as testing, but 🤷

Reply:
This point is absolutely valid. We should keep the usage and output together in the "Summary" section both for clarity and accessibility.

Having usage and output side-by-side in the initial section allows for a quick and easy reference, reducing the need to scroll through various sections to understand the impact of the code. I've removed the separate point about "output" to avoid any confusion and streamlined the section to present the expected format of the code snippet with its output. This approach aims to provide a clear and immediate understanding of what to expect from the changes.

Thank you for bringing this up!